_studio/shared/umc/io/umc_va/src/umc_va_linux.cpp

```diff
@@ -172,6 +175,17 @@ VAProfile g_VP910BitsProfiles[] =
     VAProfileVP9Profile2  // chroma subsampling: 4:2:0
 };
 
+#if defined(MFX_ENABLE_AV1_VIDEO_DECODE)
+VAProfile g_AV1Profiles[] =
+{
+    VAProfileAV1Profile0
+};
+VAProfile g_AV110BitsPProfiles[] =
+{
+    VAProfileAV1Profile1
```

Comment:
According to AV1 spec, the profile0 should support both 8bit and 10bit with 420 sampling.  Profile1 add the support for 444 sampling. So I think it is better to get 10bit profile by using profile&high_bitdepth. This may also need media-driver to support, currently media-driver code seems regard profile0 is 8bit and profile1 is 10bit.

Reply:
Where are AV1 levels defined in MediaSDK API?